tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/SimpleJNDIClientTest.java

```diff
@@ -111,22 +115,26 @@ public void testEmptyConnectionFactoryString() throws NamingException, JMSExcept
 
       //IIB v10 assumes this property is mandatory and sets it to an empty string when not specified
       props.put("java.naming.provider.url", "");
-      new InitialContext(props);//Must not throw an exception
-
+      Context ctx = new InitialContext(props);//Must not throw an exception
+      ctx.close();
    }
 
    @Test
    public void testConnectionFactoryStringWithInvalidParameter() throws Exception {
+      Level initialLevel = LogManager.getLogManager().getLogger("org.apache.activemq.artemis.core.client").getLevel();
+      LogManager.getLogManager().getLogger("org.apache.activemq.artemis.core.client").setLevel(Level.ALL);
       Hashtable<String, String> props = new Hashtable<>();
       props.put(Context.INITIAL_CONTEXT_FACTORY, "org.apache.activemq.artemis.jndi.ActiveMQInitialContextFactory");
       props.put("connectionFactory.ConnectionFactory", "tcp://localhost:61616?foo=too");
 
       AssertionLoggerHandler.startCapture();
       try {
-         new InitialContext(props);
-         assertTrue("Expected to find AMQ212078", AssertionLoggerHandler.findText("AMQ212078"));
+         Context ctx = new InitialContext(props);
+         ctx.close();
+         assertTrue("Connection factory parameter foo is not standard", AssertionLoggerHandler.findText("Connection factory parameter foo is not standard"));
```

Comment:
This assertion is failing when the test is run (and then afterwards the test sits around waiting for threads to go away).

EDIT: I now see a prior comment about actually removing the test. Not sure if there was other context around that but clearly its still here.

Reply:
The trace was moved to debug, could it be some logging misconfiguration ?

Reply:
https://github.com/apache/activemq-artemis/pull/4009/files#diff-c86b5f721ed580116525565937bfbc27f4dfcbc64c21e43a55a94780b4d90e73R20

Reply:
The code has been changed from  ActiveMQClientLogger.LOGGER.debug(String.format("Connection factory parameter %s is not standard", key)); to logger.debugf("Connection factory parameter %s is not standard", key); private static final Logger logger = Logger.getLogger(TCPSchema.class); which is a different logger category !!! this explains that

Reply:
Ah, missed that the code was changed, yep seems likely it could be related hehe.

@clebertsuconic is the fix to change the logging again, or alter the test?